portfolio/src/main/webapp/index.html

```diff
@@ -8,16 +8,30 @@
   </head>
     <header>
       <h1>JENNIFER JIMENEZ</h1>
+      <div class="logos">
+        <a href="https://www.linkedin.com/in/jen-jimenez" target="_blank"> 
+          <img src="/images/linkedin.png" id="logo"/>
+        </a>
+        <a href="https://github.com/jennifer-jimenez" target="_blank"> 
+          <img src="/images/git.png" id="logo"/>
+        </a>
+        <a href="https://www.instagram.com/prettyoddjen/" target="_blank"> 
+          <img src="/images/ig.png" id="logo"/>
+        </a>
+      </div>
     </header>
     <nav>
       <ul class="navbar">
-        <li>Home</li>
-        <li>Projects & Work</li>
-        <li>Gallery</li>
+        <li><a href="index.html" class="nav_link"><u>Home</u></a></li>
+        <li><a href="projects.html" class="nav_link">Projects</a></li>
+        <li><a href="gallery.html" class="nav_link">Gallery</a></li>
       </ul>
     </nav>
-    <body>
-    <div id="content">
+
+  <body>
+  <div id="wrapper">
```

Comment:
nit: This should be indented 2 spaces inside the `body` tag. "wrapper" is probably too generic a label to use for an `id` in most cases too. Consider using a `class` named "wrapper" if you're going to have multiple `div`s with the same styles applied to them, or possibly just removing this `div` entirely and applying the styles you want to the `body` selector.